common/caller_test.go

```diff
@@ -0,0 +1,49 @@
+package common
```

Comment:
IMO I don't think we need tests for this because runtime.Caller() and runtime.FuncForPC() is part of runtime package. I don't think we should test the runtime package in libnetwork library.